definitions/features/katello_service.rb

```diff
@@ -0,0 +1,28 @@
+class Features::KatelloService < ForemanMaintain::Feature
+  metadata do
+    label :katello_service
+  end
+
+  def make_stop(spinner, args = '')
+    originally_running_services = running_services
+    spinner.update 'Stopping katello running services..'
+    execute!("katello-service stop #{args}")
```

Comment:
the `args` could be called `services`, so that we can work with it a bit better. After doing so, we can compare the `running_services` with `services` and:

1. stop only `running_serivces & services`
2. skip this completely, if the services to be stopped are actually running.